src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -232,9 +235,21 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
     } ~
     path("admin" / "workspaces") {
       get {
+        parameters('attributeName.?, 'valueString.?, 'valueNumber.?, 'valueBoolean.?) { (nameOption, stringOption, numberOption, booleanOption) =>
           requestContext => perRequest(requestContext,
             WorkspaceService.props(workspaceServiceConstructor, userInfo),
-            WorkspaceService.ListAllWorkspaces)
+            nameOption match {
+              case None => WorkspaceService.ListAllWorkspaces
+              case Some(attributeName) =>
+                val name = AttributeName.fromDelimitedName(attributeName)
+                (stringOption, numberOption, booleanOption) match {
+                  case (Some(string), None, None) => WorkspaceService.AdminListWorkspacesWithAttribute(name, AttributeString(string))
+                  case (None, Some(number), None) => WorkspaceService.AdminListWorkspacesWithAttribute(name, AttributeNumber(number.toInt))
```

Comment:
why `toInt`? I believe AttributeNumber is backed by a BigDecimal ... so converting to int here could lose precision.